src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -233,6 +234,12 @@ export const buildATIEventTrackUrl = ({
 
   const eventPublisher = type === 'view' ? 'ati' : 'atc';
   const eventTrackingBeaconValues = [
+    {
+      key: 'idclient',
+      description: 'at user id',
+      value: getAtUserId(),
+      wrap: false,
+    },
```

Comment:
I'm sure you are aware but the same changes have been made in @jroebu14 's PR so there may be a mild conflict whichever is merged first

Reply:
Thanks for the review @andrewscfc!

Yeah I'll bear that in mind, just had to make the same changes in order to get it to work 👍